static/app/gettingStartedDocs/python/pylons.tsx

```diff
@@ -60,9 +60,9 @@ exclude_paths=my.package.crud
         description: (
           <p>
             {tct(
-              'Add the following lines to your project’s [code:.ini] file to setup [sentryHandlerCode:SentryHandler]:',
+              'Add the following lines to your project’s [initCode:.init] file to setup [sentryHandlerCode:SentryHandler]:',
```

Comment:
Was this just a typo fix?

Reply:
at first, I thought it was a typo but it also .ini in the `sentry-docs` https://raw.githubusercontent.com/getsentry/sentry-docs/master/src/wizard/python/pylons.md. It might still be a typo but will leave it as it was before